addons/test_mail/tests/test_mail_activity.py

```diff
@@ -266,6 +266,27 @@ def test_action_feedback_attachment(self):
             self.assertEqual(attachment.res_id, activity_message.id)
             self.assertEqual(attachment.res_model, activity_message._name)
 
+    def test_chained_activities(self):
+        partner_model = self.env.ref('base.model_res_partner')
```

Comment:
Could be ``Partner = self.env['ir.model']._get('res.partner')`` . That is a more common way to access models.

Reply:
ok done